static/index.html

```diff
@@ -625,7 +627,7 @@ <h3 class="logs-header" data-l10n-id="logs"></h3>
       <div id="transition-dialog-step-1">
         <img id="transition-dialog-step-1-wordmark" src="/images/webthings-lockup.svg"
              data-l10n-id="transition-dialog-wordmark" />
-        <button id="transition-dialog-close" class="close-button"></button>
+        <button id="transition-dialog-step-1-close" class="close-button"></button>
```

Comment:
Sorry I spotted this duplicate ID before and intended to fix it. This will require a CSS fix to update the selectors.

Reply:
I'll keep that in mind.

Reply:
Fixed in 599dae9908aa1e8158c5344700899f7b8d4c5c3d